test/middleware_test.rb

```diff
@@ -0,0 +1,7 @@
+require "application_system_test_case"
+
+class MiddlewareTest < ActionDispatch::IntegrationTest
+  test "error" do
```

Comment:
test renamed in https://github.com/hotwired/spark/commit/7ded7aca18a75d3d8ddea01c1b37f0364fa51fdb